M/mlpack/build_tarballs.jl

```diff
@@ -0,0 +1,175 @@
+        :libmlpack_julia_decision_tree),
+    LibraryProduct("libmlpack_julia_det", :libmlpack_julia_det),
+    LibraryProduct("libmlpack_julia_emst", :libmlpack_julia_emst),
+    LibraryProduct("libmlpack_julia_fastmks", :libmlpack_julia_fastmks),
+    LibraryProduct("libmlpack_julia_gmm_generate",
+        :libmlpack_julia_gmm_generate),
+    LibraryProduct("libmlpack_julia_gmm_probability",
+        :libmlpack_gmm_probability),
+    LibraryProduct("libmlpack_julia_gmm_train", :libmlpack_julia_gmm_train),
+    LibraryProduct("libmlpack_julia_hmm_generate",
+        :libmlpack_julia_hmm_generate),
+    LibraryProduct("libmlpack_julia_hmm_loglik", :libmlpack_julia_hmm_loglik),
+    LibraryProduct("libmlpack_julia_hmm_train", :libmlpack_julia_hmm_train),
+    LibraryProduct("libmlpack_julia_hmm_viterbi", :libmlpack_julia_hmm_viterbi),
+    LibraryProduct("libmlpack_julia_hoeffding_tree",
+        :libmlpack_julia_hoeffding_tree),
+    LibraryProduct("libmlpack_julia_kernel_pca", :libmlpack_julia_kernel_pca),
+    LibraryProduct("libmlpack_julia_kfn", :libmlpack_julia_kfn),
+    LibraryProduct("libmlpack_julia_kmeans", :libmlpack_julia_kmeans),
+    LibraryProduct("libmlpack_julia_knn", :libmlpack_julia_knn),
+    LibraryProduct("libmlpack_julia_krann", :libmlpack_julia_krann),
+    LibraryProduct("libmlpack_julia_lars", :libmlpack_julia_lars),
+    LibraryProduct("libmlpack_julia_linear_regression",
+        :libmlpack_julia_linear_regression),
+    LibraryProduct("libmlpack_julia_lmnn", :libmlpack_julia_lmnn),
+    LibraryProduct("libmlpack_julia_local_coordinate_coding",
+        :libmlpack_julia_local_coordinate_coding),
+    LibraryProduct("libmlpack_julia_lsh", :libmlpack_julia_lsh),
+    LibraryProduct("libmlpack_julia_mean_shift", :libmlpack_julia_mean_shift),
+    LibraryProduct("libmlpack_julia_nbc", :libmlpack_julia_nbc),
+    LibraryProduct("libmlpack_julia_nca", :libmlpack_julia_nca),
+    LibraryProduct("libmlpack_julia_nmf", :libmlpack_julia_nmf),
+    LibraryProduct("libmlpack_julia_pca", :libmlpack_julia_pca),
+    LibraryProduct("libmlpack_julia_perceptron", :libmlpack_julia_perceptron),
+    LibraryProduct("libmlpack_julia_preprocess_binarize",
+        :libmlpack_julia_preprocess_binarize),
+    LibraryProduct("libmlpack_julia_preprocess_describe",
+        :libmlpack_julia_preprocess_describe),
+    LibraryProduct("libmlpack_julia_preprocess_split",
+        :libmlpack_julia_preprocess_split),
+    LibraryProduct("libmlpack_julia_radical", :libmlpack_julia_radical),
+    LibraryProduct("libmlpack_julia_random_forest",
+        :libmlpack_julia_random_forest),
+    LibraryProduct("libmlpack_julia_softmax_regression",
+        :libmlpack_julia_softmax_regression),
+    LibraryProduct("libmlpack_julia_sparse_coding",
+        :libmlpack_julia_sparse_coding)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    "boost_jll",
+    "armadillo_jll",
+    "OpenBLAS_jll" # Is this necessary?
```

Comment:
I guess it is?

Reply:
I was planning on leaving this comment as a mystery for the ages, but, I guess it might be better to take it out. :smile: Done in 5e5a3d0.